src/foundry/client/apps/sidebar/apps/permission-config.d.mts

```diff
@@ -21,10 +21,14 @@ declare global {
      */
     static override get defaultOptions(): (typeof FormApplication)["defaultOptions"];
```

Comment:
Adjust the options implementation to match others, where you've got a generic with a default in the class definition at the top and then this is returning FormApplicationOptions